modules/protect.php

```diff
@@ -524,8 +524,14 @@ function kill_login() {
 		do_action( 'jpp_kill_login', $ip );
 		$help_url = 'http://jetpack.me/support/security/';
 
+		$die_string = sprintf( __( 'Your IP (%1$s) has been flagged for potential security violations.  <a href="%2$s">Find out more...</a>', 'jetpack' ), str_replace( 'http://', '', esc_url( 'http://' . $ip ) ), esc_url( $help_url ) );
+
+		if( defined('XMLRPC_REQUEST') && XMLRPC_REQUEST ) {
```

Comment:
Spaces shouldn't be necessary because it's a string not a variable.

From the wporg code standards:
$x = $foo['bar']; // correct
$x = $foo[ 'bar' ]; // incorrect

Unless it's different for parens vs. square braces, but I can't find any reference to that.